### PR TITLE
Show version in /about

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Fixes https://github.com/web-platform-tests/wpt.fyi/issues/25

## Review Information
Visit /about

### CODE REVIEW

**Comment by foolip** [comment]
https://version-dot-wptdashboard-staging.appspot.com/about shows "wpt.fyi version version" :)

**Comment by foolip** [comment]
(I'm assuming that's a bug, holding off on review.)

**Comment by lukebjerring** [comment]
Haha. Not a bug; this branch is called `version`; automatically and manually deployed versions' names will be a SHA.

**Comment by foolip** [comment]
Oh, that's a very funny non-bug :D

File: webapp/templates/about.html [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html>
-{{ template "_head.html" }}
+  <head>
+    {{ template "_head_common.html" }}
+    <style>
+      .version {
+        color: #ccc;
```
Comment (foolip) [inline webapp/templates/about.html]: This is a bit too light for me to read easily, I'll change it to `#777`.

**Pull request merged**